frontend/cypress.config.js

```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      // implement node event listeners here
+    },
```

Comment:
is this a future PR? Model Registry has this defined in their's https://github.com/kubeflow/model-registry/blob/main/clients/ui/frontend/src/__tests__/cypress/cypress.config.ts